frontend/fingertips-frontend/mock/data/healthdata.ts

```diff
@@ -2082,6 +2090,7 @@ export const mockHealthData: Record<string, HealthDataForArea[]> = {
           ageBand: '0-4',
           sex: 'Persons',
           trend: HealthDataPointTrendEnum.NotYetCalculated,
+          isAggregate: true,
```

Comment:
Again, should probably be false.
```suggestion
          isAggregate: false,
```